internal/exportTypescriptSchema.test.ts

```diff
@@ -0,0 +1,8 @@
+import { exportTypescriptSchema } from "./exportTypescriptSchema";
+
+describe("exportTypescriptSchema", () => {
+  it("exports schemas", () => {
+    const schemas = exportTypescriptSchema();
+    expect(schemas["ArrowPrimitive"]).not.toBeNull();
```

Comment:
Thanks for adding a test! What do you think of using `expect.toMatchSnapshot()` instead?

Reply:
Would we have to update the snapshot every time we add new schema?

Reply:
Yeah but we already update all the [generated files](https://github.com/foxglove/schemas/tree/main/schemas) so I don't think it's a big deal

Reply:
On the other hand, if we use this function to generate those files, then maybe having a test is redundant.

Reply:
I've updated this to use a snapshot.